[2023-02-04 10:05:00,392] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T10:00:00+00:00 [queued]>
[2023-02-04 10:05:00,400] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T10:00:00+00:00 [queued]>
[2023-02-04 10:05:00,401] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 10:05:00,401] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 10:05:00,402] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 10:05:00,418] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): bash_task> on 2023-02-04 10:00:00+00:00
[2023-02-04 10:05:00,423] {standard_task_runner.py:52} INFO - Started process 13591 to run task
[2023-02-04 10:05:00,426] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-new-dag', 'bash_task', 'scheduled__2023-02-04T10:00:00+00:00', '--job-id', '322', '--raw', '--subdir', 'DAGS_FOLDER/new_dag.py', '--cfg-path', '/tmp/tmpakcbgens', '--error-file', '/tmp/tmp5duzfrwb']
[2023-02-04 10:05:00,426] {standard_task_runner.py:80} INFO - Job 322: Subtask bash_task
[2023-02-04 10:05:00,535] {task_command.py:371} INFO - Running <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T10:00:00+00:00 [running]> on host 39df186e1726
[2023-02-04 10:05:00,620] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-new-dag
AIRFLOW_CTX_TASK_ID=bash_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T10:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-04T10:00:00+00:00
[2023-02-04 10:05:00,622] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-02-04 10:05:00,623] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'Hello, World!'"]
[2023-02-04 10:05:00,634] {subprocess.py:85} INFO - Output:
[2023-02-04 10:05:00,636] {subprocess.py:92} INFO - Hello, World!
[2023-02-04 10:05:00,637] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-04 10:05:00,674] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-new-dag, task_id=bash_task, execution_date=20230204T100000, start_date=20230204T100500, end_date=20230204T100500
[2023-02-04 10:05:00,718] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 10:05:00,764] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
